src/Bicep.Core.UnitTests/Utils/OutputHelper.cs

```diff
@@ -71,7 +71,12 @@ public static string GetDiagLoggingString(string sourceText, string outputDirect
             {
                 message = Regex.Replace(message, @"'\${TEST_OUTPUT_DIR}.*?'", new MatchEvaluator((match) => match.Value.Replace('\\', '/')));
             }
-            return $"[{diagnostic.Code} ({diagnostic.Level})] {message} |{spanText}|";
+
+            var docLink = diagnostic.Uri == null
+                ? string.Empty
+                : $"{diagnostic.Source}({diagnostic.Uri})";
+
+            return $"[{diagnostic.Code} ({diagnostic.Level})] {message} {docLink}|{spanText}|";
```

Comment:
Please could you format source & uri in a way that it's obvious they aren't part of the message to someone looking through the baseline file?

Reply:
Made a change, hope it is what you are looking for.